frontend/src/components/VLoadMore.vue

```diff
@@ -26,11 +30,18 @@ export default defineComponent({
     VButton,
   },
   setup() {
+    const loadMoreButtonRef = ref(null)
     const i18n = useI18n()
     const mediaStore = useMediaStore()
     const searchStore = useSearchStore()
 
-    const isFetching = computed(() => mediaStore.fetchState.isFetching)
+    const { sendCustomEvent } = useAnalytics()
+
+    // Use the `_searchType` from mediaStore because it falls back to ALL_MEDIA
+    // for unsupported search types.
```

Comment:
Does this matter for the load more button? We don't show it on unsuppoorted media types, right? So perhaps it'd be better to get the searchType from the search store.

Reply:
This appeases Typescript 😆